### PR TITLE
remove -webkit- references

### PR DESCRIPTION
**Pull request by argyleink**
see userland comment https://twitter.com/simevidas/status/1154960320487727105?s=20

having -webkit- in the code examples creates more questions than it helps answer

### CODE REVIEW

**Comment by mfriesenhahn** [comment]
Should we remove the reference to the `-webkit-` prefix from the copy in the second para as well?

**Comment by argyleink** [comment]
i felt the reference in the 2nd paragraph was fine since it's contextually correct due to the "recent years" reference. it's also not code that someone may copy/paste 🤷‍♂ 

tldr; it's fine as a historical reference, but shouldnt be in our code blocks

**Comment by mfriesenhahn** [comment]
> i felt the reference in the 2nd paragraph was fine since it's contextually correct due to the "recent years" reference. it's also not code that someone may copy/paste 🤷‍♂
> 
> tldr; it's fine as a historical reference, but shouldnt be in our code blocks

Cool, just wanted to double-check.

**Comment by simevidas** [comment]
If possible, add a note about using Autoprefixer to generate the prefixes, since they are still needed for Safari. 

Ideally, this would become a global guideline for all articles on web.dev, but that’s a request for the person(s) in charge of the site 😅

**Comment by argyleink** [comment]
i think the web.dev spirit would be to write a post on autoprefixer: the what, the why and the how. that way it can stand alone and be referenced. perhaps you want to write one!? this is an open blog you can submit to.

**Comment by robdodson** [comment]
Might be worth adding a quick note in the meantime, something like:

```
{% Aside %}
Some older browsers require a vendor prefixed version of this property. You can use a tool like [Autoprefixer](https://css-tricks.com/autoprefixer/) to automatically add these prefixes to your CSS.
{% endAside %}
```
